models/issues/issue_stats.go

```diff
@@ -33,6 +33,7 @@ const (
 	FilterModeMention
 	FilterModeReviewRequested
 	FilterModeReviewed
+	FilterModeSearch
```

Comment:
I don't think it should be a filter mode since this filter can combind with other filters. It's just standalone filters.

Reply:
we need some way to tell `getUserIssueStats` func that poster/assignee filter is applied, otherwise it will put doerID in everything and the count(open/closed/total) will be incorrect

Reply:
even if we pass opts with correct data, it wont apply without the filtermode
https://github.com/go-gitea/gitea/blob/main/routers/web/user/home.go#L792-L806